docs/pages/access-controls/access-graph/gitlab.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Discover GitLab Access Patterns with Teleport Policy
+description: Describes how to syncronize GitLab access patterns using Teleport Policy and Access Graph.
+---
+Teleport Policy streamlines and centralizes access management across your entire infrastructure, allowing you to view
+access relationships within seconds. You can see unified, up-to-date relationships and policies between all users, groups,
+and computing resources.
+
+With Teleport Policy's Access Graph, you gain insights into access patterns within your GitLab account. By scanning all
+permissions, users, groups, and projects, it provides a visual representation to help enhance the permission model within
+your GitLab environment. This functionality enables you to answer queries such as:
+
+- What projects are accessible to users?
+- Which users have write permissions to projects?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
+
+## How it works
+
+Access Graph synchronizes various GitLab resources, including users, projects and groups.
+These resources are then visualized using the graph representation detailed in the
+[Access Graph page](../teleport-policy/getting-started-policy.mdx).
+
+The importing process involves two primary steps:
+
+### Polling GitLab APIs
+
+The Teleport cluster continuously scans the configured GitLab accounts and retrieves the following resources:
+
+- Users
+- Groups
+- Projects
+- Group memberships
+- Project memberships
+
+Once all the necessary resources are fetched, Teleport pushes them to the
+Access Graph, ensuring that it remains updated with the latest information from your GitLab instance.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources imported and their relationships, crafting a
+graphical representation thereof.
+
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v14.3.20/v15.3.1/v16.0.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.21.4 or later.
+Check [Access Graph page](../teleport-policy/getting-started-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable
+from Teleport Auth Service.
```

Comment:
We should make it clear somehow that it's only for self-hosted and if you're using Cloud you don't have to do anything other than purchase it (unless it's a trial).